MIGRATION.md

```diff
@@ -1,5 +1,9 @@
 # Migration Guide
 
+## Migrate to `___`
```

Comment:
Version number is missing

Reply:
The version number is still missing here on purpose. It is not yet certain whether the next release will be `2.4.6` or `2.5.0`. Before I push the corresponding tag the migration guide and changelog will be adjusted accordingly. The entry here is more a reminder...